stdpopsim/drosophila_melanogaster.py

```diff
@@ -77,16 +77,40 @@ class Comeron2012_dm6(genetic_maps.GeneticMap):
 
 class SheehanSongThreeEpoch(models.Model):
     """
-    The three epoch model estimated for African samples from Sheehan and Song
+    Model Name:
+        SheehanSongThreeEpoch
 
-    .. todo:: document this model, including the original publications
-        and clear information about what the different population indexes
-        mean.
+    Model Description:
+        The three epoch (modern, bottleneck, ancestral) model estimated for a
+        single African Drosophila Melanogaster population from `Sheehan and Song <https:/
+        /doi.org/10.1371/journal.pcbi.1004845>`_ . Population sizes are estimated by a
+        deep learning model trained on simulation data. NOTE: Due to differences in
+        coalescence units between PSMC (2N) and msms (4N) the number of generations were
+        doubled from PSMC estimates when simulating data from msms.
```

Comment:
I think we should say in the note "Due to differences in
        coalescence units between PSMC (2N) and msms (4N) the number of generations were
        doubled from PSMC estimates when simulating data from msms in the original publication. We have faithfully represented the published model here"

Reply:
I just want to make sure this doesn't sound like a bug that we have injected for no reason...

Reply:
I agree. I figured more experienced word-smithing was called for for this :)